python/pyspark/sql/tests.py

```diff
@@ -558,6 +558,18 @@ def check_datatype(datatype):
         _verify_type(PythonOnlyPoint(1.0, 2.0), PythonOnlyUDT())
         self.assertRaises(ValueError, lambda: _verify_type([1.0, 2.0], PythonOnlyUDT()))
 
+        schema = StructType().add("key", LongType()).add("val", PythonOnlyUDT())
```

Comment:
This should be in its own test method - it's no longer merely a `test_udt` but rather a `test_simple_udt_in_df`.